packages/core/admin/admin/src/pages/Marketplace/components/NpmPackageCard.tsx

```diff
@@ -98,23 +101,19 @@ const NpmPackageCard = ({
                     defaultMessage: 'Plugin verified by Strapi',
                   })}
                 >
-                  <Flex>
-                    <Icon as={CheckCircle} marginLeft={2} color="success600" />
-                  </Flex>
+                  <CheckCircle fill="success600" />
```

Comment:
to confirm "marginLeft={2}" not required?

Reply:
Not anymore, since we apply the `gap` above! :)